experimenter/experimenter/jetstream/models.py

```diff
@@ -261,6 +261,9 @@ def append_conversion_count(self, primary_metrics_set):
                     primary_metric_data, BranchComparison.ABSOLUTE
                 )
 
+                if not absolute_primary_metric_vals.all:
+                    continue
+
```

Comment:
I am not sure by having this if and continue, how the below 2 statements will affect counts? may be I don't have enough context, just trying to understand 🤔